src/main/kotlin/com/cognifide/gradle/aem/common/NotifierFacade.kt

```diff
@@ -62,10 +64,18 @@ class NotifierFacade private constructor(private val aem: AemExtension) {
         return DorkboxNotifier(aem, configurer)
     }
 
-    fun custom(notifier: (title: String, text: String, level: LogLevel) -> Unit): Notifier {
+    fun custom(notifier: (title: String, text: String, level: LogLevel, clickActionHandler: (Notify) -> Unit) -> Unit): Notifier {
         return object : Notifier {
-            override fun notify(title: String, text: String, level: LogLevel) {
-                notifier(title, text, level)
+            override fun notify(title: String, text: String, level: LogLevel, onClick: (Notify) -> Unit) {
+                notifier(title, text, level, onClick)
+            }
+        }
+    }
+
+    fun notifyLogError(title: String, message: String, file: URI) {
+        notifier.notify(title, message, LogLevel.INFO) {
+            if (Desktop.isDesktopSupported() && Desktop.getDesktop().isSupported(Desktop.Action.BROWSE)) {
```

Comment:
be aware that someone could forget to configure `aem.notificationEnabled=false` and this should not break the build on CI which will not have any GUI

Reply:
OK, that's why there is this testing - additionally, this is lambda which will never get executed without clicking on the balloon.